llama_stack/templates/meta-reference-gpu/build.yaml

```diff
@@ -4,6 +4,10 @@ distribution_spec:
   description: Use Meta Reference for running LLM inference
   docker_image: null
   providers:
+    post_training:
```

Comment:
I think we should make experimental templates until new APIs settle. For now, you can just make a new `experimental-post-training` template. we will keep making backward incompatible changes otherwise and cause unnecessary breakage to our users.